src/kOS/Communication/ProcessorConnection.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using kOS.Safe.Encapsulation;
+using kOS.Module;
+using kOS.Safe.Exceptions;
+using kOS.Suffixed.PartModuleField;
+using kOS.Safe.Communication;
+
+namespace kOS.Communication
+{
+    [kOS.Safe.Utilities.KOSNomenclature("Connection", KOSToCSharp = false)]
+    public class ProcessorConnection : Connection<kOS.SharedObjects>
+    {
+        private kOSProcessor processor;
+
+        public override bool Connected {
+            get {
+                return IsCpuVessel();
+            }
+        }
+
+        public override double Delay {
+            get {
+                return IsCpuVessel() ? 0 : -1;
+            }
+        }
```

Comment:
(See elsewhere in the PR comments for the comment with the word "curly brace" in big letters.)